todo_app/data/boards.py

```diff
@@ -0,0 +1,32 @@
+class Board():
```

Comment:
These classes don't seem to be used anywhere, is that right? We should use them to fill in our templates, e.g. the `aList['cards']` could be a list of `Items` rather than a list of the Json objects returned by Trello

(It would also be nice to separate them into their own classes, for any that we do use)